feature-libs/product-configurator/rulebased/occ/variant/variant-configurator-occ.adapter.ts

```diff
@@ -324,6 +324,12 @@ export class VariantConfiguratorOccAdapter
     );
     //no need to work with a converter here, as Configurator.Variant is a projection of the OCC
     //variant representation
-    return this.http.get<Configurator.Variant[]>(url);
+    return this.http.get<Configurator.Variant[]>(url, { context: this.prepareSendUserContext() });
+  }
+
+  prepareSendUserContext(): HttpContext {
```

Comment:
I propose to rename it, what about 
```suggestion
  indicateSendUserForAsm(): HttpContext {
```